sklearnex/dispatcher.py

```diff
@@ -75,6 +76,10 @@ def get_patch_map():
                                           'NearestNeighbors',
                                           NearestNeighbors_sklearnex), None]]
 
+        mapping['local_outlier'] = [[(neighbors_module,

```

Comment:
"lof", "local_outlier_factor" ?
@Pahandrovich, @vlad-nazarov, what would you say? Are there some rules for this mapping?

Reply:
I think we don't have any rules for mapping. I guess `lof` would be better. It is better to use abbreviations if they are well known.

Reply:
Good suggestion! But I also want to notice that for all algorithms we have alias at this map with python class name in lower case. It was the cause of unpatch failures for new knn patching. Maybe is it better to add such an alias here for users who wants to unpatch this class for some reason?